test-cases/longevity/longevity-encryption-at-rest-200GB-6h.yaml

```diff
@@ -27,12 +27,3 @@ authenticator_password: cassandra
 authorizer: 'CassandraAuthorizer'
 
 pre_create_schema: True
-scylla_encryption_options: "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'LocalFileSystemKeyProviderFactory', 'secret_key_file': '/etc/scylla/encrypt_conf/secret_key'}"
```

Comment:
Why was this removed?

Reply:
it was moved to the configuration file it belongs, `local-ear.yaml`